### PR TITLE
fix(@wallet): open sign modal only when ready

### LINKED ISSUE #13071: "This is your signing phrase" popup shows during loading screen
If "Wallet" section was the last section opened, then during next app startup the "This is your signing phrase" popup appears during the loading screen:

Screenshot 2023-12-20 at 16 17 53

### PR DESCRIPTION
**Pull request by alaibe**
fixes #13071

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

File: ui/main.qml [hunk 1]
```diff
@@ -367,6 +367,7 @@ StatusWindow {
 
     Component {
         id: app
+
```
Comment (Khushboo-dev-cpp) [inline ui/main.qml:370]: can we revert this change?

File: ui/app/AppLayouts/Wallet/WalletLayout.qml [hunk 2]
```diff
@@ -44,7 +54,7 @@ Item {
     }
     
     function showSigningPhrasePopup(){
-        if(!hideSignPhraseModal && !RootStore.hideSignPhraseModal){
+        if(!hideSignPhraseModal && !RootStore.hideSignPhraseModal && visible && appMainVisible){
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/WalletLayout.qml:57]: we should add a check if popup is already `opened` too? so that we dont call popup again and again.

File: ui/app/AppLayouts/Wallet/WalletLayout.qml [hunk 3]
```diff
@@ -24,8 +24,18 @@ Item {
...
 
-    onVisibleChanged: resetView()
+    onAppMainVisibleChanged: showSigningPhrasePopup()
+
+    onVisibleChanged: {
+        resetView()
+        showSigningPhrasePopup()
+    }
+
+    Component.onCompleted: {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/WalletLayout.qml:36]: is this really needed?

**Review by Khushboo-dev-cpp — COMMENTED** [review]
Some comments, otherwise looks good

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM!
